Frontend/src/components/TableEditors/Rentals/EditRentalPopup.svelte

```diff
@@ -73,13 +73,25 @@
   export let createNew = false;
   let updateStatusOnWebsite = true;
 
-  if (createNew) {
-    doc.rented_on = new Date().getTime();
+  function returnToday() {
+    let today = new Date().valueOf();
+    doc.returned_on = today;
```

Comment:
Can you help me out here? Why is ` doc.returned_on` not updated, even though it is bound to "timeMillis" later?

Reply:
The update part is working as you intended. There is just one bug occuring when there is nor previously selected value. This was caused by the DateInput component, should be fixed now.